cli/internal/cmd/upgradeapply_test.go

```diff
@@ -83,6 +83,15 @@ func TestUpgradeApply(t *testing.T) {
 			wantErr:           true,
 			stdin:             "no\n",
 		},
+		"abort, rollback terraform err": {
```

Comment:
nit: restore seems like a more intuitive name since we use that name in the code. let's just use one of either words